### ISSUE TITLE
when filtering, total amount is not adusting

### ISSUE THREAD

**Issue by benoaa** [rgriebl/brickstore#653]
total amount should take in consideration the filters applied

**Comment by ZZJHONS** [comment]
No, total is the whole file, if you want to see the filtered totals just do a Ctrl+A

**Comment by benoaa** [comment]
when i do the filtering then ctrl+A, total remains for the whole file

Le jeu. 5 janv. 2023, à 03 h 32, Sergio S. G. ***@***.***> a
écrit :
